feature/default_sets.go

```diff
@@ -96,7 +96,9 @@ var defaultSetDesc = setDesc{
 		SetInvoice: {}, // I
 	},
 	lnwire.SimpleTaprootOverlayChansOptional: {
-		SetInit:    {}, // I
+		SetInit: {}, // I
+	},
+	lnwire.ExperimentalEndorsementOptional: {
```

Comment:
we still want `SimpleTaprootOverlayChansOptional` to be in the nodeAnn though

Reply:
Whoop, rebase gremlins!